### DISCUSSION TITLE
Is there a way to set icon size via theme?

### DISCUSSION THREAD

**Discussion by mitya33** [tusen-ai/naive-ui#4911] [Q&A]
So far as I can see the Icon component doesn't have a theme var to control its `size` prop, so you have to pass this manually to each instance of the component. Or am I missing something? I'd like to do (in theme overrides):

```
...
Icon: {
    size: '20px'
}
...
```

**Comment by jahnli** [comment]
From the theme editing, it appears that the `size` variable was not exported

**Reply by mitya33** [reply]
Thanks. Ah, that's a shame.
